docs/docs/config/bluetooth.md

```diff
@@ -16,3 +16,4 @@ See [Configuration Overview](index.md) for instructions on how to change these s
 | `CONFIG_ZMK_BLE_EXPERIMENTAL_FEATURES` | bool | Aggregate config that enables both `CONFIG_ZMK_BLE_EXPERIMENTAL_CONN` and `CONFIG_ZMK_BLE_EXPERIMENTAL_SEC`.                                                                                                                                                            | n       |
 | `CONFIG_ZMK_BLE_PASSKEY_ENTRY`         | bool | Enable passkey entry during pairing for enhanced security. (Note: After enabling this, you will need to re-pair all previously paired hosts.)                                                                                                                           | n       |
 | `CONFIG_BT_GATT_ENFORCE_SUBSCRIPTION`  | bool | Low level setting for GATT subscriptions. Set to `n` to work around an annoying Windows bug with battery notifications.                                                                                                                                                 | y       |
+| `CONFIG_ZMK_BLE_FAST_SWITCHING`        | bool | Enables faster switching between profiles by maintaining the bluetooth connection even after switching to a different profile. Disabling this will disconnect (but remain paired) from the old profile when selecting a new one.                                        | y       |
```

Comment:
```suggestion
| `CONFIG_ZMK_BLE_FAST_SWITCHING`        | bool | Enable faster switching between profiles by maintaining the bluetooth connection even after switching to a different profile. Disabling this will disconnect (but remain paired) from the previous profile when selecting a different one.                                        | y       |
```
Note that this table needs reformatting (with prettier, would be easiest, e.g. via [`npm run prettier:format`](https://zmk.dev/docs/development/documentation#formatting-and-linting-your-changes) or through the [pre-commit hooks](https://zmk.dev/docs/development/pre-commit).

Reply:
Done.